examples/encfs/aci-arm-template.json

```diff
@@ -94,8 +94,7 @@
         "imageRegistryCredentials": [
           {
             "server": "<registry-name>",
-            "username": "",

```

Comment:
I wouldn't completely remove this since it's an option. Maybe just reference the options in the README?